src/agent.h

```diff
@@ -16,6 +16,8 @@
 
 #ifndef NVAGENT_H
 #define NVAGENT_H
+#ifndef NVAGENT_VERSION
+#define NVAGENT_VERSION
```

Comment:
I wonder here if we should just spit an error like this example:
#ifndef DEBUG
#error Only Debug builds are supported
#endif

to make sure we don't build the client without a version number

Reply:
ahhh gotcha gotcha good idea will make the fix